### ISSUE TITLE
I couldn't get the project to build properly. - Only using this as there is no message option on here.

### ISSUE THREAD

**Issue by DevTony72** [DigitalEaglesManchester/digitaleagles#1]
Something must have upset Gradle from the switch of locations. Anyway Android Studio works on other projects so its all good. I've looked into it and i have a good solid idea about what it will take to make this project work in code. I will try to explain tomorrow how we will have to implement things like navigation, processing in the background, saving and connecting to a database etc. I'll make a few notes to help you with some of the coding in Android. See you tomorrow.

**Comment by JRascagneres** [comment]
Ok, I managed to get it working on my laptop after Isaac pushed but it took some time - You can maybe try again tomoro

**Comment by DevTony72** [comment]
yeah i will, i think its because he used a different SDK version. do you know which it is?

**Comment by JRascagneres** [comment]
Its 4.0

**Comment by DevTony72** [comment]
ahh okay, I've got 5.1.1 on my phone and 4.4 on my mac, wasn't sure which he went with. Thanks

**Comment by JRascagneres** [comment]
On your mac get 4.0 - When you test on your phone it will still work

**Comment by DevTony72** [comment]
All i can find is 4.0.3, i tried SDK manager, ill try it anyway

**Comment by JRascagneres** [comment]
Yeah in the SDK manager you should be able to download 4.0.0 i think

**Comment by mhutti1** [comment]
any build works but you have to change the target skd in your manifest

**Comment by mhutti1** [comment]
you have to change it in 2 locations

**Comment by JRascagneres** [comment]
I know but its easier if we all use the same one

**Comment by mhutti1** [comment]
No i mean to get it working your target sdk in your android manifest has to be one you have installed and one that works on your phone

**Comment by JRascagneres** [comment]
Yeah but i have lollipop too and I can run it - Its backwards compatible

**Comment by mhutti1** [comment]
yeah but i had a problem where it was 5.1.1 which i dont have yet and so i had to install the 5.0.1 sdk and change the settings in the manifest

**Comment by DevTony72** [comment]
Never mind, ill try it tomorrow, its not even being recognised as a Gradle project. My other projects work fine.

**Comment by JRascagneres** [comment]
Hrm, odd

**Comment by DevTony72** [comment]
By the way i just discovered that on some versions of android you are able to download an offline version of the speech recognition pack

**Comment by mhutti1** [comment]
Nice sounds good.

**Comment by JRascagneres** [comment]
That could be a secondary little thing to add

**Comment by DevTony72** [comment]
Yeah or in case there is network trouble at the presentation.

**Comment by DevTony72** [comment]
is anyone else getting a Network Error when they try to push the listen button?

**Comment by mhutti1** [comment]
No mine works fine

**Comment by DevTony72** [comment]
ill try and re-sync

**Comment by JRascagneres** [comment]
What error is it

**Comment by DevTony72** [comment]
or can you re-push your working one

**Comment by DevTony72** [comment]
07-28 18:10:23.926  11179-11179/eu.mhutti1.digitaleagles.digitaleagles I/VoiceRecognitionActivity﹕ onRmsChanged: -2.12

07-28 18:10:23.931  11179-11179/eu.mhutti1.digitaleagles.digitaleagles D/VoiceRecognitionActivity﹕ FAILED Network error

**Comment by mhutti1** [comment]
Scorp has some changes not pushed on his map border

**Comment by DevTony72** [comment]
thats ok, i just need the voice recognition to work

**Comment by mhutti1** [comment]
well im working on some stuff and will push it once it works.

**Comment by DevTony72** [comment]
Why does stuff just mess up once i get home aha. Anyway, i'll just try and write a few classes that should make it easier for you to gather all the data and store it.

**Comment by mhutti1** [comment]
well the voice recognition should work... I have got it to do it live now and am working on getting it to continuously run without restarting but it looks like google may have deliberately sabotaged the code.

**Comment by DevTony72** [comment]
Look up Android Services, maybe you can implement it in a service. Services have an options to enable a process to continuously run. Check this under the Stick vs Non-Sticky title ... http://developer.xamarin.com/guides/android/application_fundamentals/services/part_1_-_started_services/

**Comment by mhutti1** [comment]
it might work im looking it up now but it seems google may have deliberately stoped the code from working since jelly bean but kept the code there

**Comment by DevTony72** [comment]
I think my problems may be because i have CyanogenMod OS on my phone, it could be causing issues with it.

**Comment by mhutti1** [comment]
k
